examples/nat/main/Makefile

```diff
@@ -6,6 +6,6 @@ PATH_TO_MK = ../../../mk
 IMAGENAME = yanff-nat
 EXECUTABLES = nat
 
-nat: ../config.go ../translation.go ../portalloc.go
+nat: ../config.go ../translation.go ../portalloc.go ../cksum.go ../util.go
```

Comment:
No file with name util.go added in this commit

Reply:
Sorry, forgot to add it. It is fixed in pull request #111